camera/MultiCameraApplication/java/com/intel/multicamera/IntelCamera.java

```diff
@@ -0,0 +1,40 @@
+package com.intel.multicamera;

```

Comment:
- rename the file to MultiCamera or someother name.
- Add copyright and license header